src/wasm/wasm-binary.cpp

```diff
@@ -723,8 +723,8 @@ void WasmBinaryBuilder::readUserSection(size_t payloadLen) {
 
 uint8_t WasmBinaryBuilder::getInt8() {
   if (!more()) throwError("unexpected end of input");
-  if (debug) std::cerr << "getInt8: " << (int)(uint8_t)input[pos] << " (at " << pos << ")" << std::endl;
-  return input[pos++];
+  if (debug) std::cerr << "getInt8: " << (int)(uint8_t)input.at(pos) << " (at " << pos << ")" << std::endl;
+  return input.at(pos++);
```

Comment:
why is this change here? seems offtopic for the PR; also we already have a check right above it that throws an error.

Reply:
Right, sorry. My reverting of this change in #2001 was not backported to this branch. Fixing.